alarm.graphqls

```diff
@@ -37,6 +38,32 @@ input AlarmTag {
     value: String
 }
 
+type Event {
```

Comment:
Please share the object in `event.graphqls`. You don't need this duplicate one.

Reply:
Can I apply the `Event` of event.graphqls directly to alarm.graphqls?  Sorry, I'm not familiar with this query protocol.
Like this? 
![image](https://user-images.githubusercontent.com/36784140/116808054-5b809000-ab69-11eb-993b-3c253e565bf9.png)

Reply:
Yes, you could use it directly. And more importantly, you should try the latest version in your OAP, and make sure it is suitable(compilable and runnable). There are a lot of verifications when you boot the OAP server.

Reply:
OK, Thanks a lot.